src/static/reducers/cards.js

```diff
@@ -52,6 +65,48 @@ export default function cardsReducer(state = initialState, action) {
                 isCreatingCard: false
             });
 
+        case CARD_DELETE_REQUEST:
+            return Object.assign({}, state, {
+                isDeletingCard: true
+            });
+
+        case CARD_DELETE_FAILURE:
+            return Object.assign({}, state, {
+                isDeletingCard: false
+            });
+
+        case CARD_DELETE_SUCCESS:
+            return Object.assign({}, state, {
+                isDeletingCard: false
+            });
+
+        case CARD_EDIT_REQUEST:
+            return Object.assign({}, state, {
+                isEditingCard: true
+            });
+
+        case CARD_EDIT_FAILURE:
+            return Object.assign({}, state, {
+                isEditingCard: false
+            });
+
+        case CARD_EDIT_SUCCESS:
+            return Object.assign({}, state, {
+                isEditingCard: false
+            });
```

Comment:
If you do not want any difference between cases, you can group them
But it depends about what you want to do in the future

Reply:
OK